int32 senderID
int64 planID
int64 parentState
int64 planType
int32 authority
EntryPointRobots[] entrypoints